package/network/services/uhttpd/files/uhttpd.init

```diff
@@ -180,6 +180,12 @@ start_instance()
 	for listen in $http; do
 		 procd_append_param command -p "$listen"
 	done
+	
+	config_get_bool mdns "$cfg" mdns 1
+	if [ "$mdns" -ne 0 ]; then
```

Comment:
$http can be a list and can even contain duplicate ports. please make the code honor this.

Reply:
Unfortunately there is an inherent limit of 1 advertisement per `procd` service, where attempts to advertise other ports will overwrite the JSON object created previously. Given this information I have chosen to use the last valid port from the `listen_http` configuration options. 

If you find this compromise unacceptable, I understand and will close the PR.